src/main/scala/com/gu/salesforce/messageHandler/MessageHandlerLambda.scala

```diff
@@ -56,13 +39,13 @@ trait MessageHandler extends Logging {
   }
 
   def sendToQueue(notification: ContactNotification): Future[Try[SendMessageResult]] = {
-    val queueMessage = QueueMessage(notification.getSObject.getId)
+    val queueMessage = QueueMessage(notification.sObject.Id.get)
     val queueMessageString = Json.prettyPrint(Json.toJson(queueMessage))
     queueClient.send(queueName, queueMessageString)
   }
 
-  def processNotifications(notifications: List[ContactNotification], outputStream: OutputStream) = {
-    val contactListStr = notifications.map(_.getSObject.getId).mkString(", ")
+  def processNotifications(notifications: Seq[ContactNotification], outputStream: OutputStream) = {
+    val contactListStr = notifications.map(_.sObject.Id.get).mkString(", ")
```

Comment:
Note that for some reason the [Salesforce WSDL](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_steps_generate_wsdl.htm) says that the `Id` field is nillable:

https://github.com/guardian/salesforce-message-handler/blob/15d2d514a0bfb63917fd42241a294e0b58e24e16/src/main/wsdl/salesforce-outbound-message.wsdl#L34

...so `scalaxb` respects this, and makes the `Id` field an `Option` - this ends up being one of the few cases where I think it's probably reasonable to just call `.get` on an `Option`.